packages/dds/map/src/interfaces.ts

```diff
@@ -241,6 +241,19 @@ export interface IDirectoryEvents extends IEvent {
     (event: "disposed", listener: (
         target: IEventThisPlaceHolder,
     ) => void);
+
+    /**
+     * Emitted when this previously deleted sub directory is restored.
+     * This event only needs to be handled in the case of rollback. If your application does
+     * not use the local rollback feature, you can ignore this event.
```

Comment:
Can you update the comment for subDirCreated to indicate that if a delete is rolled back, this is the event emitted?